content/blog/nielx/2023-10-28_haiku_developer_tools_update_october_2023.md

```diff
@@ -0,0 +1,91 @@
++++
+type = "blog"
+title = "Haiku Developer Tools Update (October 2023)"
+author = "nielx"
+date = "2023-10-28 10:35:19+01:00"
+tags = ["haiku", "software"]
++++
+
+This article lists some of the recent updates for developer tools that are available in the HaikuPorts applications. Many of these can be installed the HaikuDepot.
+
+This is the first article on this topic, but it may become a regular (quarterly?) series. Let me know in the comments what you find of these notes, and what else you think should be covered.
+
+<!--more-->
+
+## GCC 13.2.0
+
+The Haiku nightlies are now built with GCC 13.2.0 as the primary compiler and it is available for software developers. It can also be used on a Haiku R1 beta 4 system. The previous version was GCC 11.3.0. The packages support **C, C++, and Fortran.**
```

Comment:
```suggestion
The Haiku nightlies are now built with GCC 13.2.0 as the primary compiler (the previous version was GCC 11.3.0) and it is available for software developers. It can also be used on a Haiku R1 beta 4 system. The packages support **C, C++, and Fortran.**
```